telemetry/definitions/commonDefinitions.json

```diff
@@ -402,6 +402,11 @@
             "type": "int",
             "description": "How many lines of code being sent for security scan"
         },
+        {
+            "name": "codewhispererCodeScanProjectBytes",
+            "type": "double",
```

Comment:
why double instead of int? does "int" map to a smaller type?

is kilobytes a more appropriate unit (though ideally all of the "size" types would use the same unit)?

Reply:
the main reason is that customer's project can be very large, unlike truncated payload size; and yes, we want to keep the same unit for size related fields